### DISCUSSION TITLE
Plugin candidates for @metalsmith org adoption

### DISCUSSION THREAD

**Discussion by webketje** [segmentio/metalsmith#395] [General]
3rd party plugins are flagged as candidates for adoption when all of the conditions match:

- no significant development has occurred by the maintainer for over a year
- the plugin provides a feature common to most web projects (think bundling JS, minification, image processing)
- an issue has been created at the owner's repo, which the owner has not replied to for over 3 weeks.

Currently, the following candidates have been identified:

- [metalsmith-gzip](https://github.com/ludofischer/metalsmith-gzip): [proposal issue](https://github.com/ludofischer/metalsmith-gzip/issues/15) (2023-07-17) - would become `@metalsmith/gzip`
- ~~[metalsmith-lunr](https://github.com/CMClay/metalsmith-lunr): [proposal issue](https://github.com/CMClay/metalsmith-lunr/issues/18) (2023-09-05) - would become `@metalsmith/search-index`~~. Closed due to being out of date. A `@metalsmith/search-index` plugin will be built from scratch.
- [metalsmith-html-minifier](https://github.com/whymarrh/metalsmith-html-minifier) - would become `@metalsmith/html-minifier`

**Comment by webketje** [comment]
metalsmith-gzip owner has updated the lib! :+1:

**Comment by rludgero** [comment]
@metalsmith/html-minifier

@metalsmith/tags

It would be nice to add it to the official repository, since it has not been updated for years.

Thank you for the excellent work you are doing @webketje

**Comment by webketje** [comment]
@wernerglinka has created a (somewhat opnionated) alternative to metalsmith-html-minifier, https://github.com/wernerglinka/metalsmith-optimize-html to overcome its vulnerability issues. There should be an official one but I'm short on time already doing major upgrades with existing core/plugins

For tags, I believe https://github.com/webketje/metalsmith-taxonomy is a more powerful alternative, and work is ongoing on @metalsmith/collections to make it more flexible.

**Comment by rludgero** [comment]
> @wernerglinka has created a (somewhat opnionated) alternative to metalsmith-html-minifier, https://github.com/wernerglinka/metalsmith-optimize-html to overcome its vulnerability issues. There should be an official one but I'm short on time already doing major upgrades with existing core/plugins

👍 

> For tags, I believe https://github.com/webketje/metalsmith-taxonomy is a more powerful alternative, and work is ongoing on @metalsmith/collections to make it more flexible.

I will take a look, thank you very much for the tips. 🥇
